### ISSUE TITLE
Figma cleanup

### ISSUE THREAD

**Issue by vykster** [cal-itp/benefits#560]

**Comment by srhhnry** [comment]
- [ ] Clean up screens and ensure they are all current
- [ ] Rename individual screens to ensure they are current
- [ ] Rename pages for clarity
- [ ] Create archive
- [ ] Create section for new design system screens

**Comment by srhhnry** [comment]
@machikoyasuda Just started playing around with the Measure plugin on Figma and it's starting to make sense. I could see starting to do annotations based on those measurements as well. We should chat about what measurements are the most useful

**Comment by srhhnry** [comment]
@machikoyasuda I added recaptcha to the mobile screens and while the behavior isn't great, I don't think it's a huge problem for now. With the addition of the stacked footer, at 14px from the bottom the recaptcha doesn't block the action button, which I think is the most important thing here. Added screens to show.  *Important question*---does this also appear on desktop, and if so, where on the screen?

![Image](https://user-images.githubusercontent.com/5394204/171435811-723f7253-3ed7-4891-b8b3-138d4a3115f3.png)

**Comment by machikoyasuda** [comment]
@srhhnry Captcha shows up on the pages with forms: The Enrollment page. 

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/171450410-8a8a15d6-30ed-441f-9b22-2af6f909eab4.png">

(This is a screenshot from production, so it does not reflect the redesigned page)

**Comment by srhhnry** [comment]
Alternatively, does the badge float 14px above the device viewing area? Or is it attached to the bottom of whatever content is loaded on the page? On mobile we can have users scroll for the footer, which would push the badge down away from the action buttons, if its anchored to content. If the badge floats and is always on screen, I think we might want to go with the hidden option if possible. (If not possible then I'll dig a bit more into either pushing it up or switching it to the left side.)

**Comment by machikoyasuda** [comment]
> Okay, did some digging, and have we explored hiding the badge? Found this on [google's site](https://developers.google.com/recaptcha/docs/faq):
> 
> ![Image](https://user-images.githubusercontent.com/5394204/171667018-13f87d2f-35d0-4b83-bd3f-f9cfeb3a7c26.png)
> 
> Hiding the badge is definitely preferable, since its only purpose seems to be to alert users that there is spam protection. I don't think we need that with our users and I can see the badge being very disruptive (especially on mobile).

This is great! I had no idea. This seems preferable. We would have to add text to the form pages.

**Comment by srhhnry** [comment]
Yup! The extra text is the tradeoff. I'll mock it up, but I think it'll be okay.

**Comment by thekaveman** [comment]
\+ we need this text translated, but I'm fine with this direction!

I might suggest "This _form_ is protected..." instead of "This _site_ is protected..." to be more accurate? Not sure it matters.

**Comment by srhhnry** [comment]
Yeah, I think that makes sense. It's a deviation from the Google request, but I don't think that matters too much.
